tasks/components/layout/TwoColumn.tsx

```diff
@@ -141,7 +145,7 @@ export const TwoColumn = ({
             className={tw(`overflow-hidden`)}
             style={{ width: leftWidth + 'px' }}
           >
-            <SimpleBarReact ref={scrollableRefLeft} style={{ overflowX: 'hidden', maxHeight: simpleBarMaxHeight }}>
```

Comment:
why do you removed overflowX: 'hidden'?

Reply:
Because it causes the width calculation to be off and with the corrected version the width should always match